src/t8_schemes/t8_default/t8_default_pyramid/t8_dpyramid_bits.c

```diff
@@ -91,16 +91,22 @@ compute_type_ext (const t8_dpyramid_t *p, const int level,
   return type;
 }
 
+/* TODO: Comment*/
```

Comment:
You shouldn't add any new TODOs to a PR. ;) Please resolve this.
You probably do not have to heavily document this routine anyway since it
is quite short and self-explanatory.